src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  &__star {
+    background-image: url(../images/star.svg);
```

Comment:
Ensure that the image `star.svg` exists in the `../images/` directory and the path is correct.